src/main/java/io/nats/client/NKey.java

```diff
@@ -32,13 +30,16 @@
 import static io.nats.client.support.Encoding.base32Encode;
 import static io.nats.client.support.RandomUtils.PRAND;
 import static io.nats.client.support.RandomUtils.SRAND;
+import static io.nats.nkey.NKeyConstants.*;
```

Comment:
The constants were not public in the original version, to they were extracted to the library and referenced from it.